### ISSUE TITLE
Syntax coloring doesn't recognize quoted strings in parameter validation attribute

### ISSUE THREAD

**Issue by adbertram** [PowerShell/EditorSyntax#27]
This issue was noticed in VS Code with the PowerShell plugin.
### System Details
- Operating system name and version: MacOS Sierra
- VS Code version: 1.5.3
- PowerShell extension version: 0.7.2
- Output from `$PSVersionTable`:

```
Name                           Value                                                                                                                                                                                            
----                           -----                                                                                                                                                                                            
PSVersion                      6.0.0-alpha                                                                                                                                                                                      
PSEdition                      Core                                                                                                                                                                                             
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                                          
BuildVersion                   3.0.0.0                                                                                                                                                                                          
GitCommitId                    v6.0.0-alpha.9                                                                                                                                                                                   
CLRVersion                                                                                                                                                                                                                      
WSManStackVersion              3.0                                                                                                                                                                                              
PSRemotingProtocolVersion      2.3                                                                                                                                                                                              
SerializationVersion           1.1.0.1      
```
### Issue Description

![image](https://cloud.githubusercontent.com/assets/4306938/19440989/9756eeda-9449-11e6-94f9-8a24697a3c11.png)

**Comment by ITMicaH** [comment]
I have the same issue:
![image](https://cloud.githubusercontent.com/assets/12932660/26513293/046178e0-426b-11e7-825d-431d93b37f42.png)
![image](https://cloud.githubusercontent.com/assets/12932660/26513307/1e84de74-426b-11e7-82ee-aaa53d2a2b83.png)
![image](https://cloud.githubusercontent.com/assets/12932660/26513327/3996c31c-426b-11e7-9003-88e1b98d3189.png)

**Comment by omniomi** [comment]
Fixed by #94
